module/Activity/src/Activity/Form/ActivitySignup.php

```diff
@@ -58,6 +65,23 @@ public function initialiseExternalForm($fields)
         $this->initialiseForm($fields);
     }
 
+    public function initialiseExternalForm($fields)
+    {
+        $this->add([
+            'name' => 'captcha',
+            'type' => 'Zend\Form\Element\Captcha',
+            'options' => [
+                //'label' => 'Beep-boop I\'m a robot.',
+                'captcha' => new ImageCaptcha([
+                    'font' => getcwd() . '/public/fonts/bitstream-vera/Vera.ttf',
```

Comment:
Also, I'm not sure whether this is zend framework issue or not, but some captcha's aren't really readable. 
![screenshot from 2017-01-11 16-20-52](https://cloud.githubusercontent.com/assets/7968933/21854216/4003c60c-d81a-11e6-8c07-219d89be4f32.png)
as you can see in the image, the last two characters fall outside of the captcha